doc-content/enterprise-only/bpmn/bpmn-end-events-ref.adoc

```diff
@@ -23,17 +26,21 @@ h|End
 |image:BPMN2/bpmn-end-error.png[]
 
 |<<_compensation_end_event>>
-|
+|image:BPMN2/bpmn-end-compensation.png[]
 
 |<<_escalation_end_event>> 
-|
+|image:BPMN2/bpmn-end-escalation.png[]
 
 |<<_terminate_end_event>>
 |image:BPMN2/bpmn-end-terminate.png[]
 
 
 |===
 
+[[_none_end_event]]
+.None
+
+The None End Event specifies that is no other special behavior associated with the end of the process.
```

Comment:
Probably a missing word here? that `there` is no other...

Reply:
Fixed